src/components/icon/test/icon.e2e.ts

```diff
@@ -4,10 +4,49 @@ import { test } from '@utils/test/playwright';
 test.describe('icon: basic', () => {
   test('should not have visual regressions', async ({ page }) => {
     await page.goto(`/`);
-    
+
     // Wait for all SVGs to be lazily loaded before taking screenshots
     await page.waitForLoadState('networkidle');
 
     expect(await page.screenshot({ fullPage: true })).toMatchSnapshot(`icon-diff.png`);
   });
-});
\ No newline at end of file
+
+  test('some icons should flip when rtl', async ({ page }) => {
+    await page.goto(`/`);
+
+    const autoflip = page.locator('.auto-flip-chevrons [name=chevron-forward] .icon-inner');
+    const unflip = page.locator('.un-flip-chevrons [name=chevron-forward] .icon-inner');
+    await expect(autoflip).not.toHaveCSS('transform', /matrix\(-1/);
+    await expect(unflip).not.toHaveCSS('transform', /matrix\(-1/);
+
+    await page.evaluate(() => {
+      document.dir = 'rtl';
+    });
+
+    await expect(autoflip).toHaveCSS('transform', /matrix\(-1/);
+    await expect(unflip).not.toHaveCSS('transform', /matrix\(-1/);
+
+    // Wait for all SVGs to be lazily loaded before taking screenshots
+    await page.waitForLoadState('networkidle');
+
+    expect(await page.screenshot({ fullPage: true })).toMatchSnapshot(`icon-rtl-diff.png`);
```

Comment:
Can we take screenshots of the specific icons we want to test rather than the entire page? In this case we can probably screenshot just the `.auto-flip-chevrons` and `.un-flip-chevrons` containers.

Reply:
I think we should snapshot them all. Some should flip and some should not flip, and we should validate that nothing changes that would cause e.g. all of them to flip. I suppose I could change it to only capture the "RTL" section of the page, but I do think we should capture all of those.

Reply:
Sure that works 👍